src/store/createStore.ts

```diff
@@ -93,6 +93,9 @@ export interface Store {
       bipPath: string;
       hardWalletSigs: xdr.DecoratedSignature[] | [];
     };
+    submit: {
+      xdrBlob: string;
+    };
```

Comment:
I think we should use the `xdr` store we already have. Lab v1 shows XDR in both places when added on either page, so that would be consistent with that behavior. Also, XDR can get very long, and we might run out of URL space if we need to keep 2 XDRs in the URL (for View XDR and Submit Txn).

Reply:
good point!